google_analytics/ga_homepage_report.py

```diff
@@ -421,13 +426,18 @@ def output_report(filename, total_homepage_views=None, total_course_card_clicks=
         featured_cards_worksheet.write('B6', int(featured_cards['uniqueClicks'].sum()), comma_fmt)
         featured_cards_worksheet.write('B7', total_course_card_clicks, comma_fmt)
         featured_cards_worksheet.write('B8', total_program_card_clicks, comma_fmt)
-        featured_cards_worksheet.write('B10', float(featured_cards['uniqueClicks'].sum())/total_homepage_views, percent_fmt)
+        featured_cards_worksheet.write('B10', float(featured_cards['uniqueClicks'].sum()) / total_homepage_views,
+                                       percent_fmt)
         featured_cards_worksheet.write('B13', int(featured_cards['uniqueEnrolls'].sum()), comma_fmt)
         featured_cards_worksheet.write('B14', int(featured_cards['uniqueCourseEnrolls'].sum()), comma_fmt)
         featured_cards_worksheet.write('B15', int(featured_cards['uniqueProgramEnrolls'].sum()), comma_fmt)
-        featured_cards_worksheet.write('C13', float(featured_cards['uniqueEnrolls'].sum())/total_homepage_views, percent_fmt)
-        featured_cards_worksheet.write('C14', float(featured_cards['uniqueCourseEnrolls'].sum()) / total_homepage_views, percent_fmt)
-        featured_cards_worksheet.write('C15', float(featured_cards['uniqueProgramEnrolls'].sum()) / total_homepage_views, percent_fmt)
+        featured_cards_worksheet.write('C13', float(featured_cards['uniqueEnrolls'].sum()) / total_homepage_views,
+                                       percent_fmt)
+        featured_cards_worksheet.write('C14', float(featured_cards['uniqueCourseEnrolls'].sum()) / total_homepage_views,
+                                       percent_fmt)
+        featured_cards_worksheet.write('C15',
+                                       float(featured_cards['uniqueProgramEnrolls'].sum()) / total_homepage_views,
+                                       percent_fmt)
```

Comment:
Would it make sense to do some of these calculations above the method invocation and pass them as named variables instead?